src/analyses/goto_check.cpp

```diff
@@ -174,11 +174,19 @@ class goto_checkt
 
   std::string array_name(const exprt &);
 
+  /// Include the \p asserted_expr in the code conditioned by the \p guard.
+  /// \param asserted_expr: expression to be asserted
+  /// \param comment: human readable comment
+  /// \param property_class: classification of the property
+  /// \param source_location: the source location of the original expression
+  /// \param src_expr: the original expression to be included in the comment
+  /// \param guard: the condition under which the asserted expression should be
+  ///   taken into account
   void add_guarded_claim(
```

Comment:
While we are here, can we rename `claim` to `property` since we have replaced that terminology in other places in the codebase?

Reply:
Done.